### PR TITLE
docs: Improve some documentation

### PR DESCRIPTION
**Pull request by antonhalim**
**What**:
Improve some documentation in Readme.md

**Why**:

**How**:

**Checklist**:

- [ ] Documentation added to the
      [docs site](https://github.com/testing-library/testing-library-docs)
- [ ] Tests
- [ ] Typescript definitions updated
- [ ] Ready to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,8 +156,8 @@ afterAll(() => {
 // hidden-message.js
 import React from 'react'
 
-// NOTE: React Testing Library works with React Hooks _and_ classes just as well
-// and your tests will be the same however you write your components.
+// NOTE: React Testing Library works well with React Hooks and classes.
+// Your tests will be the same whichever you write your components.
```
Comment (nickmccurdy) [inline README.md:160]: I don't think `whichever` is grammatically correct here 

```suggestion
// Your tests will be the same however you write your components.
```

Comment (hdn7) [reply README.md:160]: Maybe `whichever` can be replaced with `regardless of how` 👀

Comment (antonhalim) [reply README.md]: Thanks for the input, its updated. cc @nickmccurdy

**Review by nickmccurdy — APPROVED** [review]

**Comment by nickmccurdy** [comment]
@add-contributors add @antonhalim for docs

**Comment by nickmccurdy** [comment]
@all-contributors add @antonhalim for docs

**Pull request merged**